libbeat/reader/syslog/message.go

```diff
@@ -177,14 +183,16 @@ func (m message) fields() common.MapStr {
 	f := common.MapStr{}
 
 	// Syslog fields.
-	_, _ = f.Put("log.syslog.priority", m.priority)
-	_, _ = f.Put("log.syslog.facility.code", m.facility)
-	_, _ = f.Put("log.syslog.severity.code", m.severity)
-	if v, ok := mapIndexToString(m.severity, severityLabels); ok {
-		_, _ = f.Put("log.syslog.severity.name", v)
-	}
-	if v, ok := mapIndexToString(m.facility, facilityLabels); ok {
-		_, _ = f.Put("log.syslog.facility.name", v)
+	if m.prioritySet {
```

Comment:
I'd replace all these _,_ with a single nolint on the if statements

Reply:
If you're referring to removing the explicit discards on the return values, I'd rather keep them in place. While it is more verbose, it makes it more clear to the reader that return values are being ignored here.